### PR TITLE
Version formatting edits addressing Issue #135

### PR DESCRIPTION
**Pull request by attp**
- remove version footers (and associated files)
 - fix image headers

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -23,7 +23,7 @@ We have organized the tutorials into three sections:
 Since quantum computing is new to most users, we want to find the best *Hello Quantum World* program and welcome submissions here. 
 
 ### 2. [Reference](reference/)<a id='reference'></a>
```
Comment (rraymondhp) [inline README.md]: Thanks. This looks good! But, do we need to say version 0.5.3? Because I think we are sure it will move forward quickly.

Comment (attp) [reply README.md]: I've changed all mentions of 0.5.3 to 0.5, dropping the minor version number.

**Comment by jaygambetta** [comment]
once you resolve the conflict i am good with this.

**Comment by jaygambetta** [comment]
addresses issue #135

**Pull request merged**